### PR TITLE
edits to our blog introduction of dvol

### PR DESCRIPTION
**Pull request by Stephenitis**

### CODE REVIEW

File: blog.md [hunk 1]
```diff
@@ -1,53 +1,62 @@
-# Docker Voluminous plugin from ClusterHQ
+# Making the (bridging the gap/making the leap/easing the transition) development to production
```
Comment (wallnerryan) [inline blog.md]: I like keeping this to just one line, ```#Easing the transition of development to production```, have (something/something/something) is hard to read.

File: blog.md [hunk 2]
```diff
@@ -1,53 +1,62 @@
...
+# ALT: The Path to Docker in Production
+# ALT: dvol: Paving the road to Docker in Production
```
Comment (wallnerryan) [inline blog.md]: +1 to the name in the title, and the title being useful.

File: blog.md [hunk 3]
```diff
@@ -1,53 +1,62 @@
...
-At ClusterHQ we started out making stateful containers work in production with Flocker.
+At ClusterHQ we are laser focused on making stateful containers a reality for developers and teams ready to make the jump into production.
 
-We are also interested in how stateful containers can be useful for developers in development environments.
+Before making the jump to production with stateful containers Developers should first have confidence in the tools they are testing in their development and testing environments. To help ease the stress of making this leap our team has been in the labs working on a set of tools that will help increase confidence and reduce the cycle of waiting for test and debugging failures.
 
-So we asked ourselves the question: what do developers spend most of their time doing?
+We had some clear goals with this tool
+- makes it faster to develop with docker
+- improve the development user experience
```
Comment (wallnerryan) [inline blog.md]: I feel like its odd state isn't mentioned here. Like ```improve the development user experience when working with state/"stateful-ness"```

File: blog.md [hunk 4]
```diff
@@ -129,6 +171,6 @@ TODO
 
 ## Reference: implementation
 
-Voluminous volumes, with the default plain filesystem driver, consist of a branches, which is a directory of files in `/var/lib/dvol/volumes/<volumename>/branches/<branchname>`, and a set of commits in `/var/lib/dvol/volumes/<volumename>/commits/<id>`, which are simply copies of those directories.
+dvol volumes, with the default plain filesystem driver, consist of a branches, which is a directory of files in `/var/lib/dvol/volumes/<volumename>/branches/<branchname>`, and a set of commits in `/var/lib/dvol/volumes/<volumename>/commits/<id>`, which are simply copies of those directories.
```
Comment (wallnerryan) [inline blog.md]: s/consist of a branches/consists of branches/

Comment (wallnerryan) [inline blog.md]: This also left me wondering, what other filesystems does ```dvol``` work with? e.g. ```dvol volumes, with the default plain filesystem driver,```

File: blog.md [hunk 5]
```diff
@@ -55,59 +64,93 @@ docker run --rm -ti -v /var/lib/dvol:/var/lib/dvol \\
...
 
-You can start Docker containers with Voluminous volumes:
+## How do I use it?
 
+- *dvol CLI* : primary user interface for dvol, init, commit, reset, branch, etc.
+- *dvol Docker*: Volume Plugin* quickly connect your containers to your dvol volumes. `--volume-driver=dvol`
+- *dvol volumes*: like Docker containers, are global to your development machine.
+
+#### 1. init a new volume
+$ `dvol init frob_mysql`
```
Comment (wallnerryan) [inline blog.md]: this suggests somthing with mysql, yet below we're running busybox. Does dvol working with all databases flocker/docker supports? If so, lets mention this somewhere

File: blog.md (hunk 1 above)
Comment (Stephenitis) [inline blog.md]: @wallnerryan agreed, I was just putting all the ideas i had in brain for a title. @lukemarsden (discussed in office)  wants to more accurately frame this not about going towards production but more about making the development experience faster/better.

File: blog.md (hunk 3 above)
Comment (Stephenitis) [inline blog.md]: +1

**Comment by lukemarsden** [comment]
Sorry for clobbering this without considering the feedback in this PR. These are some great ideas in here, especially making clear that "dvol work[s] with all databases flocker/docker". @wallnerryan could you propose a PR against the new README? Thanks!

**Pull request closed without merging**